commands/releases/output.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+let cli = require('heroku-cli-util')
+let co = require('co')
+let request = require('request')
+
+function * run (context, heroku) {
+  function getRelease (id) {
+    return heroku.get(`/apps/${context.app}/releases/${id}`)
+  }
+
+  function getLatestRelease () {
+    return heroku.request({
+      path: `/apps/${context.app}/releases`,
+      partial: true,
+      headers: { 'Range': 'version ..; max=2, order=desc' }
+    }).then((releases) => releases[0])
+  }
+
+  let release
+  if (context.args.release) {
```

Comment:
As I understand this branching, it'll be invoked if `heroku releases:output --release 123` is passed in. Can we make it so it aligns with `heroku release:info 123`? I believe it can be made so that the positional argument is optional.